lib/modules/apostrophe-doc-type-manager/views/chooserChoices.html

```diff
@@ -9,15 +9,21 @@
     {% include data.choiceTemplate %}
     {% if not item.readOnly %}
       <div class="apos-button apos-button--in-context apos-button--group apos-chooser-controls">
-        {{ buttons.inGroup('', { icon: 'arrow-up', action: 'raise-item', value: choice._id}) }}
-        {{ buttons.inGroup('', { icon: 'arrow-down', action: 'lower-item', value: choice._id }) }}
+{# <<<<<<< HEAD
+        {{ buttons.inGroup('', { icon: 'arrow-up', action: 'raise-item', value: choice._id, tooltip: 'Move Up'}) }}
+        {{ buttons.inGroup('', { icon: 'arrow-down', action: 'lower-item', value: choice._id, tooltip: 'Move Down' }) }}
+        {{ buttons.inGroup('', { icon: 'trash', action: 'delete-item', value: choice._id, tooltip: 'Trash' }) }}
+======= #}
+        {{ buttons.inGroup('', { icon: 'arrow-up', action: 'raise-item', value: choice._id, tooltip: 'Move Up'}) }}
+        {{ buttons.inGroup('', { icon: 'arrow-down', action: 'lower-item', value: choice._id, tooltip: 'Move Down' }) }}
+{# >>>>>>> 0d26a18d097857b78b41b0dc36d5d1c3e6b9fe4e #}
```

Comment:
Another merge conflict hanging out

Reply:
ha thanks tom, this was underbaked, working on it

Reply:
Bump - a year plus later (: Any interest in resolving these issues and landing this?